selfdrive/hardware/base.h

```diff
@@ -6,8 +6,8 @@
 // no-op base hw class
 class HardwareNone {
 public:
-  static constexpr float MAX_VOLUME = 0;
-  static constexpr float MIN_VOLUME = 0;
+  static constexpr float MAX_VOLUME = 1.0;
+  static constexpr float MIN_VOLUME = 0.5;
```

Comment:
let's lower this a bit, since I expect this will be more for debugging than in car use
```suggestion
  static constexpr float MAX_VOLUME = 0.7;
  static constexpr float MIN_VOLUME = 0.3;
```